core/templates/core/index.html

```diff
@@ -3,21 +3,36 @@
 {% block title %}Strona Główna{% endblock %}
 
 {% block content %}
+    <nav class="bg-white border-gray-200 dark:bg-gray-900">
+        <div class="flex flex-wrap justify-between items-center mx-auto max-w-screen-xl p-4">
+            <a href="https://github.com/Gwardia-Czapli/GwardiaHub" class="hover:text-blue-100 text-gray-400 transition-colors duration-300" target="_blank">
+                <i class="fa-brands fa-github fa-3x mr-5"></i>
+                <span class="text-4xl text-white font-bold my-5">GwardiaHub</span>
+            </a>
+        </div>
+    </nav>
+    <nav class="bg-gray-50 dark:bg-gray-700">
```

Comment:
What's the purpose of this second nav?

Reply:
Well, then better way would be creating div for each icon-link pair, with flex-col, because rn icons aren't aligned on one line with its link. Tho there is no need for that because we don;t need icons, only links